unl/UNL_Events
# Implement DCF Date Picker

Testing:
* Verify removal of jquery-ui does not break anything
* Verify all old date picker CSS is removed.

For both event create and edit:
* Verify date pickers for start, end and recurring end work and display correctly
* Verify the monthly recurring options populate correctly. Note: On start date change will not work until DCF update is applied to WDN.  You can verify this if testing locally with current 5.3 WDN.  The `This is a recurring event` checkbox will trigger the monthly options update to test on non up to date WDN.
